File: app/src/main/java/com/gnardini/tvshowcontroller/TvShowControllerApp.java
package com.gnardini.tvshowcontroller;

import android.app.Application;
import android.content.Context;

public class TvShowControllerApp extends Application {

    private static Context appContext;

    @Override
    public void onCreate() {
        super.onCreate();
        appContext = this;
    }

    public static Context getAppContext() {
        return appContext;
    }
}

File: app/src/main/java/com/gnardini/tvshowcontroller/TvShowControllerApp.kt
package com.gnardini.tvshowcontroller

import android.app.Application
import android.content.Context
import com.gnardini.tvshowcontroller.injector.CommonInjector

class TvShowControllerApp : Application() {

    override fun onCreate() {
        super.onCreate()
        appContext = this
    }

    companion object {

        lateinit var appContext: Context
            private set

        val commonInjector by lazy { CommonInjector() }
    }
}
